### DISCUSSION TITLE
Japanese model ja_core_news_lg training config

### DISCUSSION THREAD

**Discussion by xwd** [explosion/spaCy#8664] [Language Support]
Hi spaCyers, 

just wondering if the training config used for training the default ja model ja_core_news_lg is available for reference anywhere? I’m using the default config to train the ja NER model from scratch using the same JP-UD data, somehow, I’m only getting very low F1 scores (<30%). Any pointer to the training config would be much appreciated — just trying to pin down where the issues are. 

Thanks.

**Comment by adrianeboyd** [comment]
The config is always included with the trained model, so it's part of the `ja_core_news_lg` package. You can find it in the directory where it's installed in your python environment, usually a path like `.../venv/lib/python3.8/site-packages/ja_core_news_lg/ja_core_news_lg-3.1.0/config.cfg`.

Or you can load the model and print the config:

```python
nlp = spacy.load("ja_core_news_lg")
print(nlp.config.to_str())
```
